### DISCUSSION TITLE
Authentik in rootless docker?

### DISCUSSION THREAD

**Discussion by ser** [goauthentik/authentik#7548] [General]
Has anyone tried to run authentik in rootless environment?

**Comment by BeryJu** [comment]
It should work just fine, aside from having to disable the docker integration for outposts (or integrating with docker over TCP HTTP instead of a unix socket)

**Reply by ser** [reply]
Encouraged by your answer, I've just tried and successfully deployed authentik into a rootless docker. Thanks!
